package.json

```diff
@@ -6,11 +6,16 @@
     "type": "git",
     "url": "git://github.com/mozilla/normandy.git"
   },
+  "bin": {
+    "stylelint": "./node_modules/stylelint/dist/cli.js"
+  },
```

Comment:
I'm confused - are you suggesting we don't need this bin line at all and `stylelint` should already work without it? Or that we should swap out the `stylelint` in `lint-css` with `node_modules/.bin/stylelint`?